### PR TITLE
Book-Library

### PR DESCRIPTION
**Pull request by arikarim**
In the book-Library project, a user can create a book with a name, description, and number of pages.

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Invalid Submission

### Hi @arikarim :wave: 

I have marked this submission as invalid because there are no linter validations for the commits you made to github. Here is the [link](https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635156/) to the project requirement about setting up linters in the **remember to** section.
Also having a linter validation is a strong requirement, read through this [link](
https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-#
)

* [ ]  Ensure that you have your commits validated by the linters you have setup.

#### That's it, make this change then submit another form for a code review.

### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

**Comment by arikarim** [comment]
I am terribly sorry, I forgot to put the linters.

File: main.js [hunk 1]
```diff
@@ -1,4 +1,105 @@
...
+const library = [];
 
+const bookList = document.getElementById('book-list');
+const submitButton = document.getElementById('submit-btn');
+const form = document.querySelector('form');
+form.style.display = 'none';
 
-let form = document.querySelector(form);
+const addBook = document.getElementById('add-book');
+// addBook.style.visibility = 'visible'
```
Comment (rukundoeric) [inline main.js]: remove this commented out code line

File: main.js [hunk 2]
```diff
@@ -1,4 +1,105 @@
...
+const cancelBook = document.getElementById('close');
+// cancelBook.style.visibility = 'hidden'
```
Comment (rukundoeric) [inline main.js]: remove this commented-out code line.

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - ✔️ The app is functional 💯

- ### To Improve
  - [x] Kindly make sure you remove those two commented out code lines before you merge.

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**